src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getDurationMetadata.test.ts

```diff
@@ -0,0 +1,37 @@
+import * as assert from 'assert';
+import { assign } from 'lodash';
+
+import { getDurationMetadata } from '../../../../service/storeStatements/queriables/getMetadataFromStatement/getDurationMetadata';
+import { statementDefaults } from './fixtures/statements.fixture';
+
+describe(
+  'Retrieve duration metadata from statement',
+  () => {
+    it(
+      'should return result duration statement',
+      () => {
```

Comment:
Same as the other comment, I think we should changes occurrences of this to a single line if it doesn't exceed the line length limit in the linting rules.